### ISSUE TITLE
Something wrong in training --NaN or Inf found in input tensor.

### ISSUE THREAD

**Issue by dengxy2000** [liuyuan-pal/Gen6D#68]
Thanks for your excellent work!
When I run source code for training models on my own device(RTX 3090),something wrong with loss value,it seems not be a mistake with the dataset,could you help me with this problem?
![微信图片_20230406175655](https://user-images.githubusercontent.com/128802163/230343518-8171c982-b9aa-4dfb-9497-d3f841126f0d.jpg)
部署到另一台P5000上不会出现这种情况，是卡的问题吗？

**Comment by liuyuan-pal** [comment]
这个我也不太清楚，你要不看看是从哪一个step开始出现了NaN。
```
if torch.sum(torch.isnan(some_tensors))>0:
    import ipdb; ipdb.set_trace()
```

**Comment by dengxy2000** [comment]
> 这个我也不太清楚，你要不看看是从哪一个step开始出现了NaN。
> 
> ```
> if torch.sum(torch.isnan(some_tensors))>0:
>     import ipdb; ipdb.set_trace()
> ```

感谢你的帮助hh,问题已经解决了,我将torch的版本从1.12降低到1.10就能正常work了

**Comment by dengxy180** [comment]
你好，请问能披露一些关于refiner训练的细节吗，我在使用源码的情况下即使增加训练的total_step也未能获得与论文中相当的结果，请问你是在使用默认配置在step=260000处就能得到model_best吗？ @liuyuan-pal

**Comment by dengxy2000** [comment]
你好，还有一个想问的问题就是我在使用源码的默认参数下训练我的refiner，尽管增加total_setp到600k也无法得到与论文中相当的精度的结果，我的detector和selector应该是没啥问题的，请问你方便披露一下refiner的训练细节吗，是在默认配置下只需要260000个step就取得了best_model吗，谢谢！ @liuyuan-pal

**Comment by liuyuan-pal** [comment]
我这里有用200张cat的图片来做validation set，然后根据validation上面的performance选择一个最好的refiner。

这里在projection-2d这个metric是比较容易复现的，但是ADD那个metric随机性很大，差一两个像素就会差特别远，所以ADD不是很稳定，比较难以复现。
